plugins/bundle/plugin.go

```diff
@@ -27,6 +27,13 @@ import (
 	"github.com/open-policy-agent/opa/storage"
 )
 
+// maxActivationRetry represents the maximum number of attempts
+// to activate persisted bundles. Activation retries are useful
+// in scenarios where a persisted bundle may have a dependency on some
+// other persisted bundle. As there are no ordering guarantees for which
+// bundle loads first, retries could help in the bundle activation process.
+const maxActivationRetry = 10
```

Comment:
We normally don't recommend multiple bundles so this retry number was selected such that we can successfully activate up to 10 bundles in the worst case that they depend on each other. Also we don't want to spend too much time activating bundles that never will successfully activate.

Reply:
We could mirror some of this comment into the code comments? I think it would help to know that this is to accommodate a usage scenario we don't recommend but that can happen.

Reply:
Updated code comment to include the rationale behind choosing `10` as the maxActivationRetry value.